acme/acme_structure.go

```diff
@@ -389,7 +389,18 @@ func certDaysRemaining(cert *certificate.Resource) (int64, error) {
 	expiry := c.NotAfter.Unix()
 	now := time.Now().Unix()
 
-	return (expiry - now) / 86400, nil
+	return (expiry - now), nil
+}
+
+// certDaysRemaining takes an certificate.Resource, parses the
+// certificate, and computes the days that it has remaining.
+func certDaysRemaining(cert *certificate.Resource) (int64, error) {
+	remaining, err := certSecondsRemaining(cert)
+	if err != nil {
+		return 0, fmt.Errorf("Unable to calculate time to certificate expiry")
```

Comment:
Can we bubble up the error from certSecondsRemaining here?

It could be either just returned by itself (`return 0, err`), or as part of the format string.

Also, if including an error message, they should (almost) always be lower case. Example:

```go
return 0, fmt.Errorf("unable to calculate time to certificate expiry: %s", err)
```

Reply:
Resolved